pkg/registries/filter.go

```diff
@@ -192,7 +192,11 @@ func applyMatchSets(
 	filteredVals := []string{}
 	totalSet := toMatchSetT(totalList)
 
-	if len(whiteMatchSet) != 0 && len(blackMatchSet) != 0 {
+	if len(whiteMatchSet) == 0 {
```

Comment:
I'm concerned with this change. There is an intended use-case for specifying only a blacklist. You are saying you want to accept everything except those that don't pass my blacklist filter.

Reply:
What I could do is default the white_list to ```".*"``` if only black_list exists.